### ISSUE TITLE
Rescuezilla-2.4.1-64bit and Rescuezilla-2.4.2-64bit: unsupported feature(s): FEATURE_C12

### ISSUE THREAD

**Issue by Gitfredenpunkt** [rescuezilla/rescuezilla#413]
The following error message occurs when cloning Linux-Arch partitions. 
Failed to resize /dev/sdb2 (ext4] partition: Failed to execute command: e2fsck -f -y /dev/sdb2 
/dev/sdb2: ********** WARNING: Still file system errors **********
e2fsck 1.46.5 (30-Dec-2021)
/dev/sdb2 has unsupported property(ies): FEATURE_C12

e2fsck: Newer version of e2fsck needed!

Arch creates the partitions with the current e2fsck 1.47.0 (5-Feb-2023) !!!

**Comment by shasheene** [comment]
Thanks for the report.

OK, the filesystem check tool `e2fsck` failing to process the filesystem, with GParted Live (I believe Debian based) also failing.

As a rolling release distribution, Arch Linux has very recent packages, so it's no surprise it may create filesystems using tooling with more recently developed features, that Debian/Ubuntu based distributions like Rescuezilla cannot immediately read.

Rescuezilla currently only compiles the most recent versions of a few select packages, and that does not yet include `e2fsprogs` package, so it just uses the Ubuntu default version.

Separate note: Rescuezilla's 2.4.2's default release is still based on Ubuntu 22.04 (Jammy) (April 2022) and there is a release based on Ubuntu 22.10 (Kinetic) that may have more recent packages. But it probably suffers from this same issue.

Rescuezilla v2.5 will be coming in hopefully June after I have taken time-off work to work on the next version, but I may be able to release a version of Rescuezilla based on Ubuntu 23.04 much sooner, which is more likely to fix this issue.

**Comment by skywalker123p** [comment]
Yes there is an issue of backup and then restore of Arch Linux. When this will be fixed? thanks

**Comment by shasheene** [comment]
Does it occur on the ["Weekly" Rolling Release (2023-11-26)](https://github.com/rescuezilla/rescuezilla/releases)'s default Mantic-based release?

**Comment by skywalker123p** [comment]
Weekly Rolling release works fine.
